test/models/partner_test.rb

```diff
@@ -201,5 +202,61 @@ class PartnerAddressOrServiceAreaPresenceTest < ActiveSupport::TestCase
 
     assert @new_partner.valid? == true, 'Partner should valid'
   end
+end
 
+class PartnerAddressOrServiceAreaPermissionsTest < ActiveSupport::TestCase
+  setup do
+    @user = create(:user)
+    @user_neighbourhood = create(:neighbourhood, unit_code_value: "E05011368")
+    @user.neighbourhoods << @user_neighbourhood
+  end
+
+  test "verify: address in user ward" do
+    new_partner = build(
+      :partner, 
+      address: create(
+        :address,
+        neighbourhood: @user_neighbourhood
+      )
+    )
+
+    new_partner.accessed_by_id = @user.id
+    new_partner.save!
```

Comment:
From the fact the one set of GH actions tests failed and the other set of GH actions tests did not, it looks like it may introduce a test concurrency race condition?

```

Error:
PartnerAddressOrServiceAreaPermissionsTest#test_verify:_address_in_user_ward:
ActiveRecord::RecordInvalid: Validation failed: Partners cannot be created outside of your ward.
    test/models/partner_test.rb:224:in `block in <class:PartnerAddressOrServiceAreaPermissionsTest>'
```